dom.bs

```diff
@@ -3636,6 +3638,8 @@ that is a <a>document</a>.
 <a>node</a>'s <a>assigned slot</a>, if <a>node</a> is <a>assigned</a>, and <a>node</a>'s
 <a for=tree>parent</a> otherwise.
 
+<p class="note no-backref">Each <a for=/>node</a> also has a <a>registered observer list</a>.
```

Comment:
Each node has also ?

Reply:
The precedent in the document is "also has". And I think that's correct.

Reply:
But which one reads better?

Reply:
"also has" reads more natural to me.